app/models/change-event-type.js

```diff
@@ -63,7 +63,7 @@ export const CHANGE_EVENTS_IGS = [
   CHANGE_EVENT_TYPE.IN_ONTBINDING,
   CHANGE_EVENT_TYPE.IN_VEREFFENING,
   CHANGE_EVENT_TYPE.ONTBONDEN_EN_VEREFFEND,
-  CHANGE_EVENT_TYPE.FUSIE,
+  //CHANGE_EVENT_TYPE.FUSIE,
```

Comment:
I think we want to keep this one. The ticket only mentions removing the option for municipalities and OCMW. This block configures IGS, which is an other type of organization, and as far as I know we want to keep the fusion option for them, at least for now !